### PR TITLE
Empty message on logs table

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Empty message on logs table](https://trello.com/c/91V5oEFf/531-empty-message-on-logs-table)

File: new-lamassu-admin/src/pages/MachineLogs.js [hunk 1]
```diff
@@ -123,16 +123,21 @@ const Logs = () => {
...
-                logsResponse.machineLogs.map((log, idx) => (
-                  <TableRow key={idx} size="sm">
-                    <TableCell>{formatDate(log.timestamp)}</TableCell>
-                    <TableCell>{log.logLevel}</TableCell>
-                    <TableCell>{log.message}</TableCell>
-                  </TableRow>
-                ))}
-            </TableBody>
+            {logsResponse &&
+              (!logsResponse.machineLogs.length ? (
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/MachineLogs.js]: I think you can just check for the existence of machineLogs and not necessarily its length.
```suggestion
              (!logsResponse.machineLogs ? (
```

File: new-lamassu-admin/src/pages/ServerLogs.js [hunk 2]
```diff
@@ -174,18 +174,22 @@ const Logs = () => {
...
-                      logLevel === SHOW_ALL || log.logLevel === logLevel.code
-                  )
-                  .map((log, idx) => (
-                    <TableRow key={idx} size="sm">
-                      <TableCell>{formatDate(log.timestamp)}</TableCell>
-                      <TableCell>{log.logLevel}</TableCell>
-                      <TableCell>{log.message}</TableCell>
-                    </TableRow>
-                  ))}
+                (!data.serverLogs.length ? (
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/ServerLogs.js]: Same here
```suggestion
                (!data.serverLogs ? (
```

**Review by chaotixkilla — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/MachineLogs.js (hunk 1 above)
Comment (josepfo) [reply new-lamassu-admin/src/pages/MachineLogs.js]: You are right, I think I was missing the point of the card. I should check !logsResponse to show the empty message

**Review by chaotixkilla — APPROVED** [review]

File: new-lamassu-admin/src/pages/MachineLogs.js [hunk 3]
```diff
@@ -134,6 +134,10 @@ const Logs = () => {
                 ))}
             </TableBody>
           </Table>
+          {loading && <H4>{'Loading...'}</H4>}
+          {!loading && !R.isEmpty(logsResponse) && (
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/MachineLogs.js]: `!R.isEmpty(logsResponse)` isn't this going to show "No activity so far" only when the table has data? Also the logs are in `logsResponse.machineLogs` so the empty test should be of this.

File: new-lamassu-admin/src/pages/ServerLogs.js [hunk 4]
```diff
@@ -188,6 +188,8 @@ const Logs = () => {
                   ))}
             </TableBody>
           </Table>
+          {loading && <H4>{'Loading...'}</H4>}
+          {!loading && !R.isEmpty(data) && <H4>{'No activity so far'}</H4>}
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/ServerLogs.js]: Same as above

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**